index.d.ts

```diff
@@ -87,14 +93,15 @@ export interface MarkdownIt {
     normalizeLinkText: {(url: string): string};
     validateLink: {(url: string): string};
 
-    disable(rules: string|string[], ignoreInvalid?: boolean): MarkdownIt;
-    enable(rules: string|string[], ignoreInvalid?: boolean): MarkdownIt;
+    disable(rules: string | string[], ignoreInvalid?: boolean): MarkdownIt;
+    enable(rules: string | string[], ignoreInvalid?: boolean): MarkdownIt;
     parse(src: string, env: any): Token[];
     parseInline(src: string, env: any): Token[];
     render(src: string, env: any): string;
     renderInline(src: string, env: any): string;
     use(plugin: any): MarkdownIt;
     render(src: string): string;
+  }
 }
 
-export function MarkdownIt(preset?:string, options?:MarkdownItOptions): void;
+export = MarkdownIt;
```

Comment:
`export =` is equivalent to `module.exports =` in CommonJS. Since this is what the source code does, we want to match it so the type definition is accurate.